emojirades/commands/scorekeeper_commands/leaderboard_command.py

```diff
@@ -1,73 +1,48 @@
 import pendulum
 
 from emojirades.analytics.leaderboard import LeaderBoard
+from emojirades.analytics.time_range import TimeRange
 from emojirades.commands import BaseCommand
+from emojirades.printers.leaderboard_printer import LeaderboardPrinter
 
 
 class LeaderboardCommand(BaseCommand):
+
     description = "Shows all the users scores"
 
     patterns = (
         r"<@{me}> (score|leader)[\s]*board$",
-        r"<@{me}> (score|leader)[\s]*board (?P<range>weekly|monthly)",
+        r"<@{me}> (score|leader)[\s]*board (?P<range>weekly|monthly|alltime)",
```

Comment:
I kinda like 'all time' as 'alltime' seems a bit weird, as it's not a single word? Can we cater for both?